frontend/src/tests/modules/EmibInboxRedux.test.js

```diff
@@ -18,6 +18,8 @@ describe("EmibInboxRedux", () => {
   beforeEach(() => {
     stubbedInitialState = {
       emails: emailsJson.emailsEN,
+      emailsEN: emailsJson.emailsEN,
```

Comment:
It feels like this is going to get messy - I'd rather see :

emails: {
 english: {},
 french: {}
}

I'm also not sure I understand why we need this - let's talk in person.